PowerPointLabs/PowerPointLabs/FunctionalTestInterface.Impl/PowerPointOperations.cs

```diff
@@ -40,6 +40,33 @@ public bool IsInFunctionalTest()
             return PowerPointLabsFT.IsFunctionalTestOn;
         }
 
+        public void MaximizeWindow(int windowNumber)
+        {
+            FunctionalTestExtensions.GetApplication().Windows[windowNumber].Activate();
+            FunctionalTestExtensions.GetApplication().Windows[windowNumber].WindowState = PpWindowState.ppWindowMaximized;
+        }
+
+        public void NewWindow()
+        {
+            Presentation presentation = FunctionalTestExtensions.GetPresentations().Add();
+        }
+
+        public int GetNumWindows()
+        {
+            return FunctionalTestExtensions.GetApplication().Windows.Count;
+        }
+
+        public HashSet<Type> GetOpenPaneTypes()
+        {
+            DocumentWindow w = FunctionalTestExtensions.GetApplication().Windows[1];
+            HashSet<Type> result = new HashSet<Type>();
+            foreach (Microsoft.Office.Tools.CustomTaskPane p in FunctionalTestExtensions.GetAddIn().CustomTaskPanes)
+            {
+                if ((DocumentWindow)p.Window == w) { result.Add(p.Control.GetType()); }

```

Comment:
Let's use more descriptive variable names like `docWindow` and `pane`. While this is a relatively short function and it's easy to read, it might grow in the future it might introduce unnecessary confusion. More descriptive variables can definitely also benefit future devs that might come back to this code in the future.